iquip/apps/builder.py

```diff
@@ -301,10 +301,14 @@ class _ScanEntry(_BaseEntry):
           "selected": The name of the selected scannable type.
           "NoScan", "RangeScan", "CenterScan", and "ExplicitScan": The dictionary that contains 
             argument info of the corresponding scannable type.
-        stackWidget: The QStackedWidget that contains widgets of each scannable type.
-        rangeWidget: TODO(AIJUH): The Widget that will be removed at next PR.
+        stackedWidget: The QStackedWidget that contains widgets of each scannable type.
+        scanButtonGroup: The QButtonGroup that groups QRadiobuttons for selecting scan widget.
+        scanWidgetsDict: The dictionary that contains each scan widget.
+        scanButtonsDict: The dictionary that describes id of QRadiobutton at scanButtonGroup.
     """
-    def __init__(self, name: str, argInfo: Dict[str, Any], parent: Optional[QWidget] = None):
+    def __init__(
+        self, name: str, argInfo: Dict[str, Any], parent: Optional[QWidget] = None
+        ): # pylint: disable=too-many-locals
```

Comment:
I think this Pylint comment can be moved to the definition above.
And the definition can be written in a line.